### PR TITLE
Expand the Import History topic with field description and instruction, how to download Imported/Error file

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) expands the Import History topic with field description and instruction, how to download Imported/Error file

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/system/data-import-history.html

### CODE REVIEW

File: src/system/data-import-history.md [hunk 1]
```diff
@@ -14,3 +14,19 @@ _Import History_
...
+|Imported file| Link for downloading the imported file.|
+|Error file| The corresponding error file.|
+|Execution Time| Time interval of the import process.|
+|Summary| The number of items created, updated and deleted, or the error message.|
+
+## Download Imported/Error file
+
+1. To download _Imported/Error_ file, click the **Download** link
+
+1. Locate the export file at the downloads location for your web browser and open the file.
```
Comment (leslietilling) [inline src/system/data-import-history.md:32]:
```suggestion
## Download the Imported/Error file

1. To download the _Imported/Error_ file, click **Download**.

1. Locate the export file at the downloads location for your web browser and open it.
```

File: src/system/data-import-history.md [hunk 2]
```diff
@@ -14,3 +14,19 @@ _Import History_
...
+
+|Field|Description|
+|--- |--- |
+|ID| An nternal number used to designate a transfer.|
+|Start Date & Time| A specific date and time when the transfer took place.|
+|User| The customer who made the transfer.|
+|Imported file| Link for downloading the imported file.|
+|Error file| The corresponding error file.|
+|Execution Time| Time interval of the import process.|
+|Summary| The number of items created, updated and deleted, or the error message.|
```
Comment (leslietilling) [inline src/system/data-import-history.md:26]:
```suggestion
|Summary| The number of items created, updated, and deleted, or the error message.|
```

File: src/system/data-import-history.md [hunk 3]
```diff
@@ -14,3 +14,19 @@ _Import History_
 
 {:.bs-callout-info}
 By default, import history files are located in the `<Magento-root-directory>/var/import_history` folder. If the Remote storage module is enabled, import history files are located in the `<remote-storage-root-directory>/import_export/import_history` folder.
+
+|Field|Description|
+|--- |--- |
+|ID| An nternal number used to designate a transfer.|
```
Comment (leslietilling) [inline src/system/data-import-history.md:20]:
```suggestion
|ID| An internal number used to designate a transfer.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits

**Comment by dobooth** [comment]
@EvheniiaKolkhiieva

**Review by leslietilling — APPROVED** [review]

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**